frontend/build.mjs

```diff
@@ -50,6 +50,9 @@ await buildInParallel(
             // make sure we don't link to a global window.define
             banner: { js: 'var posthogToolbar = (function () { var define = undefined;' },
             footer: { js: 'return posthogToolbar })();' },
+            // This isn't great but we load some static assets at runtime for the toolbar and we can't sub in
+            // a variable at runtime it seems...
+            publicPath: isDev ? '/static/' : 'https://app.posthog.com/static/',
```

Comment:
need to detect app or eu?

Reply:
There isn't a way as this is at bundle time... Although I guess that could be passed in 🤔

Reply:
Nah this can't be as it happens at the docker build point. I think we can live with this for now though